packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -23,45 +23,44 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
    */
   adapter: CallAdapter;
   callInvitationURL?: string;
-
   /**
    * A callback function that can be used to provide custom data to an Avatar.
    */
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
-
   /**
    * Flags to hide UI elements of the {@link CallComposite}.
```

Comment:
Is `CallCompositeOptions` only for hiding/showing UI elements, for example could we have feature flags here also? (e.g. `newAwesomeLowerLatencyStreams: boolean`)